torch/distributions/__init__.py

```diff
@@ -29,20 +51,31 @@
     loss = -m.log_prob(action) * reward
     loss.backward()
 
-Another way to implement these stochastic/policy gradients would be to use the
-reparameterization trick from :meth:`~torch.distributions.Distribution.rsample`
-method, where the parameterized random variable can be defined as a parameterized
-deterministic function of a parameter-free random variable. The reparameterized sample
-is required to be differentiable. The code for implementing the pathwise estimation would
-be as follows::
+2. Another way to implement these stochastic/policy gradients would be to use the reparameterization
+trick via the :meth:`~torch.distributions.Distribution.rsample` method, where the random variable
+can be defined as a parameterized deterministic function of a parameter-free random variable.
+Reparameterized samples are differentiable with respect to the parameters of their distribution, and
+are limited to distributions which have the ``rsample`` method. Code for implementing the pathwise
+gradient estimate would be as follows::
 
     params = policy_network(state)
     m = Normal(*params)
-    # any distribution with .has_rsample == True could work based on the application
+    # any distribution with .has_rsample == True can work
     action = m.rsample()
     next_state, reward = env.step(action)  # Assume that reward is differentiable
     loss = -reward
     loss.backward()
+
+Advanced Features
+-----------------
+
+The ``torch.distributions`` package also implements KL divergences between many common distributions, their
+entropies, and methods for transforming distributions. For example::
```

Comment:
Is it worth mentioning about `mean` and `stddev`?

Reply:
It's probably worth mentioning above, in the Basics section. I'll make that change, thank you.